packages/experimental/Icon/src/Icon.types.ts

```diff
@@ -13,7 +13,7 @@ export interface SvgIconProps {
   uri?: string;
   src?: React.FC<SvgProps>;
   viewBox?: string;
-  color?: string;
+  color?: ColorValue;
```

Comment:
I thought we needed to keep this string because RN-SVG doesn't support ColorValue?

Reply:
Yeah, Ruriko told me that. I'll remove it